domains/foyet.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Link to my blog/personal website",
+    "repo": "https://github.com/THL007/personal-blog",
+    "owner": {
+        "username": "thl007",
+        "email": "thl7.de@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://thl007.github.io"
```

CNAME can't contain protocol, also i cant seem to view your website while visiting it
```suggestion
        "CNAME": "thl007.github.io"
```